how-to/integrate_with_cos.rst

```diff
@@ -0,0 +1,142 @@
+Integrate With Canonical Observability Stack (COS)
+==================================================
+
+This document shows how to integrate the different components of JAAS with the 
+`Canonical Observability Stack`_ to enable pre-configured dashboards and alerting rules.
+
+The Canonical Observability Stack `COS-Lite`_ is a Juju bundle that includes a series of
+open source observability applications and related automation. 
+For the complete list of components in COS, read the 
+`Component List <https://charmhub.io/topics/canonical-observability-stack/editions/lite>`__.
+
+Prerequisites
+-------------
+
+- A running `COS-Lite`_ bundle. 
+  You can follow the `Getting started on MicroK8s <https://charmhub.io/topics/canonical-observability-stack/tutorials/install-microk8s>`__
+  tutorial to get you started. Make sure to follow the section **Deploy the COS Lite** bundle with overlays sections.
+- A running JAAS. Please refer to the deployment :doc:`tutorial<../tutorial/deploy_jaas_microk8s>`.
+
+It is generally recommended to keep the observability stack separate from any observed applications to separate failure domains. 
+This document assumes that JAAS and the COS bundles are deployed to different models.
+
+This how-to assumes that Vault and PostgreSQL are deployed alongside JIMM and OpenFGA. Depending on your approach, this may not be true.
+Additionally this how-to assumes the names of the deployed applications, these might differ in your environment.
+
+Integration approaches
+----------------------
+
+There are 2 possible  integration approaches depending on your networking / deployment setup setup:
+
+1. If you are able to send metrics and logs directly to the observability platform components follow 
+   the `Integrate JAAS with COS-Lite`_ section
+2. If you prefer using a telemetry collector component follow 
+   the `Integrate JAAS with COS-Lite through Grafana-Agent`_ section
+
+Integrate JAAS with COS-Lite 
+----------------------------
+
+Grafana integration 
+^^^^^^^^^^^^^^^^^^^
+
+Assuming you deployed the COS-Lite bundle in model ``cos-model`` with user admin, use the following 
+commands to integrate the JAAS applications by means of an application offer.
+
+.. code:: bash
+
+    juju integrate jimm admin/cos-model.grafana-dashboards
```

Comment:
this also assumes that application offers were created, right? is that step described anywhere?

Reply:
Added a "hint" section on this above.